frontend/src/Frontend/Footer.hs

```diff
@@ -26,4 +28,7 @@ footer = do
       extLink "https://reddit.com/r/reflexfrp" $ text "Reddit"
       extLink "http://webchat.freenode.net?channels=%23reflex-frp&uio=d4" $ text "#reflex-frp on IRC"
   where
-    externalLinkWithTitle title url = elAttr "a" ("href" =: url <> "title" =: title)
+    externalLinkWithTitle title url m = do
```

Comment:
Seems like what we really need is just a way to slap more attrs onto the `a` tag. Perhaps `extLink'` takes that extra info and `extLink` provides `mempty` as default.

Reply:
Yeah... it's not as pretty as it could be, I agree.  There are a couple of issues along those lines around the codebase;  another is the `minWidth` tag in `Frontend.hs` should probably be moved into static css.